### PR TITLE
Dependency Updates for week of 08/15/22

### PR DESCRIPTION
**Pull request by ujahio**
### This round of updates includes:
- Passing [CI Workflow](https://app.circleci.com/pipelines/github/flexion/ef-cms/37066/workflows/75254217-ceb1-41b5-a336-bdc1c8b47762/jobs/239118) with a migration in flexion's exp5 env.

### Major package updates:
  - stylelint-config-standard
  
### Minor package updates:
  - aws-sdk

### Patch package updates:
  - artillery
  - esbuild
  - esbuild-css-modules-plugin

### Other notes

- [ ] `Terraform` updated to 1.2.7.  This will require a new docker image to be created in courts' environments (v2.3.0).
- 1 moderate audit item found with quill (a dependency of React quill) audit issues found

### CODE REVIEW

File: docs/team-process.md [hunk 1]
```diff
@@ -159,4 +158,6 @@ Below is a list of dependencies that are locked down due to known issues with se
...
     - How does the patch run?
       - This runs as part of the `npm postinstall` step. 
-    - Common troubleshooting: If you see the high severity audit issue warning for  `dicer`, run a full `npm install` rather than a single package update, as this will run the `postinstall` which is required to run the patch that addresses the security issue.
\ No newline at end of file
+    - Common troubleshooting: If you see the high severity audit issue warning for  `dicer`, run a full `npm install` rather than a single package update, as this will run the `postinstall` which is required to run the patch that addresses the security issue.
+
+3. `ajv` : temporarily installed as a project dependency due to lint:swagger failure in Github actions initiated during PR commits. The specific failure is "Cannot find module 'ajv/dist/core" failing on the lint:swagger script. `ajv` is a tranisitive dependeny of multiple packages, including swagger-cli, which seems to be causing the issue. [Link](https://github.com/ustaxcourt/ef-cms/runs/7286153454?check_suite_focus=true) to failing test on 08/15/22 dependency updates.
```
Comment (TomElliottFlexion) [inline docs/team-process.md]: Can be removed again

File: .circleci/config.yml [hunk 2]
```diff
@@ -7,10 +7,10 @@ commands:
     steps:
       - restore_cache:
           keys:
-            - v22-npm-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
+            - v23-npm-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
```
Comment (TomElliottFlexion) [inline .circleci/config.yml:10]: I think this shouldn't be necessary anymore

Comment (TomElliottFlexion) [reply .circleci/config.yml:10]: Also, I think you may have already undone it, and I reverted it (back). My bad

**Review by TomElliottFlexion — APPROVED** [review]

File: docs/team-process.md (hunk 1 above)
Comment (ujahio) [reply docs/team-process.md]: Done.

**Pull request merged**